packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3774,6 +3774,7 @@ class _UpdateTextSelectionToAdjacentLineAction<T extends DirectionalCaretMovemen
       _verticalMovementRun = currentRun;
       _runSelection = newSelection;
     }
+    return null;
```

Comment:
The return type of this method could also be void?

Reply:
done